frontend/src/App.tsx

```diff
@@ -1,26 +1,26 @@
 import React from 'react';
-import { BrowserRouter as Router, Route, Routes} from 'react-router-dom';
+import {BrowserRouter as Router, Route, Routes} from 'react-router-dom';
```

Comment:
Why do you rename your BrowserRouter? At first read I was confused what "Router" is doing at line 11.